schedule/index.html

```diff
@@ -8,18 +8,17 @@
         <h1>Conference Schedule</h1>
     </div>
 </div>
-<!--
 <div class="row white-background">
     <div class="col-xs-12 col-md-12">
         <p>
         The schedule is available as a Google Calendar in the following formats:
-        <a href="http://www.google.com/calendar/feeds/scaleconf.org_qth3oks5367pl9h6m31bhtvp2o%40group.calendar.google.com/public/basic">
```

Comment:
@vhata you own all the Google Calendars? @adrianmoisey suggested that I rather add it to the existing one - which makes sense. Can you share it with me so that I can edit it?